clang/test/CIR/CodeGen/complex.c

```diff
@@ -0,0 +1,130 @@
+// RUN: %clang_cc1 -triple x86_64-unknown-linux-gnu -fclangir -emit-cir -o - %s | FileCheck %s --check-prefixes=C,CHECK
```

Comment:
Please write files to disk, it makes it convenient to run locally and look at the output

Reply:
Updated.